src/extension.ts

```diff
@@ -895,7 +895,19 @@ export function findKindNameOrPrompt(resourceKinds: kuberesources.ResourceKind[]
 }
 
 function promptKindName(resourceKinds: kuberesources.ResourceKind[], descriptionVerb, opts, handler) {
-    vscode.window.showInputBox({ prompt: "What resource do you want to " + descriptionVerb + "?", placeHolder: 'Empty string to be prompted' }).then((resource) => {
+    let placeHolder: string = 'Empty string to be prompted';
+    let prompt: string = "What resource do you want to " + descriptionVerb + "?";
+    if (opts) {
```

Comment:
Thanks, this is the sort of thing I was thinking of.  Sorry I explained it badly.

Reply:
no worries! am glad i understood.